lib/Models/Cesium.js

```diff
@@ -423,6 +423,138 @@ Cesium.prototype.computePositionOnScreen = function(position, result) {
+    }
+    return N;
+}
+
+/**
+ * Raise an item's level in the viewer
+ * This does not check that index is valid
+ * @param {Number} index The index of the item to raise
+ */
+Cesium.prototype.raise = function(index) {
+    var items = this.terria.nowViewing.items;
+    var item = items[index];
+    var itemAbove = items[index - 1];
+
+    if (!defined(itemAbove.numItems) && !defined(itemAbove.imageryLayer)) {
+        return;
+    }
+
+    // Both item and itemAbove may either have a single imageryLayer, or be a composite item
+    // Composite items have an items array of further items.
+    // Define N as the number of subitems in ItemAbove (1 except for composites)
+    // if item is a composite, then raise each subitem in item N times,
+    // starting with the one at the top - which is the last one in the list
+    // if item is not a composite, just raise the item N times directly.
+
+    var N = numSubItems(itemAbove);
+    var i, j, subItem;
+
+    if (defined(item.items)) {
+        for (i = item.items.length - 1; i >= 0; --i) {
+            subItem = item.items[i];
+            if (defined(subItem.imageryLayer)) {
+                for (j = 0; j < N; ++j) {
+                    this.scene.imageryLayers.raise(subItem.imageryLayer);
+                }
+            }
+        }
+    }
+
+    if (!defined(item.imageryLayer)) {
+        return;
+    }
+
+    for (j = 0; j < N; ++j) {
+        this.scene.imageryLayers.raise(item.imageryLayer);
+    }
+};
+
+/**
+ * Lower an item's level in the viewer
+ * This does not check that index is valid
+ * @param {Number} index The index of the item to lower
+ */
+Cesium.prototype.lower = function(index) {
+    var items = this.terria.nowViewing.items;
+    var item = items[index];
+    var itemBelow = items[index + 1];
+    if (!defined(itemBelow.numItems) && !defined(itemBelow.imageryLayer)) {
+        return;
+    }
+
+    // same considerations as above, but lower composite subitems starting at the other end of the list
+
+    var N = numSubItems(itemBelow);
+    var i, j, subItem;
+
+    if (defined(item.items)) {
+        for (i = 0; i < item.items.length; ++i) {
+            subItem = item.items[i];
+            if (defined(subItem.imageryLayer)) {
+                for (j = 0; j < N; ++j) {
+                    this.scene.imageryLayers.lower(subItem.imageryLayer);
+                }
+            }
+        }
+    }
+
+    if (!defined(item.imageryLayer)) {
+        return;
+    }
+
+    for (j = 0; j < N; ++j) {
+        this.scene.imageryLayers.lower(item.imageryLayer);
+    }
+};
+
+/**
+ * Lowers this imagery layer to the bottom, underneath all other layers.  If this item is not enabled or not shown,
+ * this method does nothing.
+ * @param {CatalogItem} item The item to lower to the bottom (usually a basemap)
+ */
+Cesium.prototype.lowerToBottom = function(item) {
+
+    if (defined(item.items)) {
+        // TODO: think about whether this loop is better fwd or backward
```

Comment:
Try to avoid leaving TODOs in the code.  If it can't be addressed now but you don't want to forget about it, write a GitHub issue instead.